src/commonMain/kotlin/baaahs/shows/HeartbleatShow.kt

```diff
@@ -14,7 +14,7 @@ import kotlin.math.sin
 object HeartbleatShow : Show("Heartbleat") {
     override fun createRenderer(sheepModel: SheepModel, showRunner: ShowRunner): Renderer {
         return object : Renderer {
-            val beatProvider = showRunner.getBeatProvider()
+            val beatProvider = showRunner.getBeatSource()
```

Comment:
Rename `beatProvider`

Reply:
Removed, not used in this show because it just looks at showRunner directly